src/command_modules/azure-cli-monitor/azure/cli/command_modules/monitor/_help.py

```diff
@@ -30,16 +30,15 @@
             Email:   --action email bob@contoso.com ann@contoso.com
             Webhook: --action webhook https://www.contoso.com/alert apiKey=value
             Webhook: --action webhook https://www.contoso.com/alert?apiKey=value
-            To specify multiple actions, use multiple --action args.
+            Multiple actions can be specified by using more than one `--action` argument.
         - name: --description
           short-summary: Free-text description of the rule. Defaults to the condition expression.
         - name: --disabled
           short-summary: Create the rule in a disabled state.
         - name: --condition
-          short-summary: The condition expression upon which to trigger the rule.
+          short-summary: The condition which triggers the rule.
           long-summary: |
             Usage:   --condition "METRIC {>,>=,<,<=} THRESHOLD {avg,min,max,total,last} PERIOD"
-            Example: --condition "Percentage CPU > 60 avg 1h30m"
```

Comment:
This is very very deliberately placed right here. Please revert.

Reply:
Please provide a justification. Examples go down in the example sections - examples should not be needed to be included in an argument description, or if they are, they are shown as a usage example. If the argument is so complex that it requires an example to demonstrate its usage, then it would be better to provide a list of acceptable METRIC values and the format of PERIOD.

Reply:
It isn't possible--at all--to provide a list of acceptable metric values. There's a whole command for that which depends on the resource you are targeting. The reason I have an example in the parameter help text is so the user doesn't have to read the usage, and then, if still confused, scroll down to the command examples to see how to use just that one parameter. 

Usage strings are not intuitive for many so having a real example in close proximity would be more helpful. Does this cause some kind of bizarre display in the docs?

Reply:
The reason it was removed is that while there are other possibly complicated arguments to other `az` commands, none of them contain an inline example. The `--action` arguments have these as well and should be similarly removed.

If the documentation of the argument is not intuitive enough to provide an understanding without an example, that is a failing of the documentation. To me the format makes it clear what style of input is expected, but the lack of definition for METRIC, THRESHOLD, and PERIOD is what makes the example necessary. Additionally having an inline example is misleading - it may lead users to make _assumptions_ about the availability or format of these values which could be wrong.

Instead the `long-summary` here should include definitions of the various values and be structured similarly to:

> For conditions, metrics and appropriate threshold types can be retrieved with the `az monitor metrics` commands. The period for a condition is written as a timestamp in `XXhXXmXXs` format.

This, along with the existing examples, should be enough to provide adequate context.

Reply:
I'm fine with that.

Reply:
I have a followup question about PERIOD, due to a description later in the helpfile:

```
          - name: --period
            short-summary: >
              Time span over which to apply --aggregation, in nDnHnMnS shorthand or full ISO8601 format.
```

Is this an accurate description of how PERIOD works when set in conditions, as well?